my-calendar/src/java/org/sakaiproject/widgets/mycalendar/ui/WidgetPage.html

```diff
@@ -174,13 +174,15 @@
 
 /**
  * Parse the event list and count the number of events that match the date
- * @param date from the datepicker, is in the format mm/dd/yyyy
+ * Note: the way this is written it will work even if the timezone on the Sakai
+ * end is not the same as the browser's.
```

Comment:
When you say 'Sakai end' what do you mean. The timezone a user has configured in their preferences? Or the timezone when the event was created?
The timezone is passed into the page based on the Sakai Locale.